dropwizard-auth/src/test/java/io/dropwizard/auth/principal/NoAuthPrincipalEntityResource.java

```diff
@@ -13,7 +13,9 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * Contains resource methods which don't authenticate but use principal instance injection and thus might be affected by authentication logic.
```

Comment:
Please, don't reformat source code without a need. It makes changes cleaner and focused.